tests/structured_conf/data/attr_classes.py

```diff
@@ -393,15 +393,15 @@ class Str2User(Dict[str, User]):
     class Str2StrWithField(Dict[str, str]):
         foo: str = "bar"
 
-    @attr.s(auto_attribs=True)
-    class Str2IntWithStrField(Dict[str, int]):
-        foo: str = "bar"
-
     class Error:
         @attr.s(auto_attribs=True)
         class User2Str(Dict[User, str]):
             pass
 
+        @attr.s(auto_attribs=True)
+        class Str2IntWithStrField(Dict[str, int]):
+            foo: int = 1
+
```

Comment:
Why is this  under Error?
The above one is an error because it's an illegal Structured Config.
There is nothing wrong with the one you moved here.

Reply:
But this one is used to test that gives an error. I'll move it